python/lsst/rubintv/production/oneOffProcessing.py

```diff
@@ -146,6 +148,8 @@ def __init__(
 
         self.redisHelper = RedisHelper(butler, self.locationConfig)
         self.efdClient = makeEfdClient()
+        self.consdbClient = ConsDbClient("http://consdb-pq.consdb:8080/consdb")
```

Comment:
Doesn't seem great to have this hard-coded here.

Reply:
Agreed - moved to locationConfig (it might need more than that at USDF if/when that becomes something that we actually start writing to from there, but we can punt on that for now)